calebwagner/sink-repair
# created plumbers selection option

I created a section element in the Requests function to display plumber options.

Steps to check code:
1. git fetch --all
2. git checkout cjw-completion-state
3. code . 
4. check Requests.js module for section element
5. serve
6. create an order
7. select a plumber
